src/main/java/io/reactivex/internal/operators/maybe/MaybeIsEmptySingle.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ * the License for the specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivex.internal.operators.maybe;
+
+import io.reactivex.*;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.internal.disposables.DisposableHelper;
+import io.reactivex.internal.fuseable.*;
+import io.reactivex.plugins.RxJavaPlugins;
+
+/**
+ * Signals true if the source Maybe signals onComplete, signals false if the source Maybe
+ * signals onNext.
```

Comment:
Thanks, fixing it in the next PR along with more algorithmically detectable javadoc problems.